<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_employe</class>
 <widget class="QDialog" name="Dialog_employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QTabWidget::pane { /* The tab widget frame */
     border-top: 2px solid #6374a6;
     position: absolute;
     top: -0.5em;
 }
 
 QTabWidget::tab-bar {
     alignment: center;
 }
 
 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #d5ddf7, stop: 1 #87b3ff);
     border: 2px solid #6374a6;
     border-bottom-color: #6374a6; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
 }
 
 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #d5ddf7, stop: 1 #b0c2f7 );
 }
 
 QTabBar::tab:selected {
     border-color: #6374a6;
     border-bottom-color: #6374a6; /* same as pane color */
 }</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Ajouter un employé</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>661</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QGroupBox {
    
	background-color: rgb(255, 255, 255);
     margin-top: 1ex; /* leave space at the top for the title */
 }
 </string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="la_matricule">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string>Matricule</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_nom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_prenom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_poste">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string>Poste</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_datenaissance">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_salaire">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>250</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_matricule">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_salaire">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_poste">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_datenaissance">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>10</y>
        <width>781</width>
        <height>451</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>background.png</pixmap>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>la_matricule</zorder>
     <zorder>la_nom</zorder>
     <zorder>la_prenom</zorder>
     <zorder>la_poste</zorder>
     <zorder>la_datenaissance</zorder>
     <zorder>la_salaire</zorder>
     <zorder>le_matricule</zorder>
     <zorder>le_salaire</zorder>
     <zorder>le_prenom</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_poste</zorder>
     <zorder>le_datenaissance</zorder>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_matricule">
     <property name="geometry">
      <rect>
       <x>535</x>
       <y>383</y>
       <width>119</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QComboBox {
     border: 1px solid #bac4e0;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
 }
 
 QComboBox:editable {
     background: #bac4e0;
 }
 
 QComboBox:!editable, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #d5ddf7, stop: 1 #b0c2f7);
 }
 
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 23px;
 
     border-left-width: 1px;
     border-left-color: #bac4e0;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
 
 QComboBox::down-arrow {
     image: url(C:/Users/youss/Desktop/ESPRIT/Projet C++/smart_station/icon.png);
 }
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_nettoyer">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Nettoyer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Afficher la liste des employés</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>15</y>
       <width>531</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #d5ddf7;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_trim">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Tri matricule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tris">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Tri salaire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_trip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Tri poste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_pdf">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>Exporter PDF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>-10</y>
       <width>701</width>
       <height>471</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>background.png</pixmap>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>tableView</zorder>
    <zorder>pb_trim</zorder>
    <zorder>pb_tris</zorder>
    <zorder>pb_trip</zorder>
    <zorder>pb_pdf</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Chercher un employé</string>
    </attribute>
    <widget class="QLineEdit" name="le_recherche">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>11</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>12</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: #000000;
background:#dce3f7;
border: #6374a6;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}</string>
     </property>
     <property name="text">
      <string>Matricule</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>461</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #d5ddf7;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_chat">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	 color: #1549a3;
     border: 2px solid #bac4e0;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #d5ddf7, stop: 1 #b0c2f7);
     min-width: 80px;
 }
 
 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #b0c2f7, stop: 1 #d5ddf7);
 }
</string>
     </property>
     <property name="text">
      <string>chat</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-21</y>
       <width>671</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QGroupBox {
     
	background-color: rgb(255, 255, 255);
     margin-top: 1ex; /* leave space at the top for the title */
 }</string>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>15</y>
        <width>671</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>background.png</pixmap>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_3</zorder>
    <zorder>le_recherche</zorder>
    <zorder>label</zorder>
    <zorder>tableView_2</zorder>
    <zorder>pb_chat</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
